app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,28 @@
+<header class="main-header">
+    <div class="header-wrapper">
+      <div class="main-logo">
+        <%= link_to "#", class: "navbar-brand" do %>
+          <i class="fas fa-leaf"></i> Reuseful
+        <% end %>
+      </div>
+      <nav>
+        <ul class="main-menu">
+          <li><a href="#section-2">About</a></li>
+          <li><a href="#section-7">Story</a></li>
+          <li><a href="#section-9">FAQs</a></li>
+          <li><a href="#main-footer">Contact</a></li>
+          <% if @user %>
```

Comment:
Shouldn't we use if current_user here instead?

Reply:
yea, I was waiting for you guys to tell me what the variable name is

Reply:
because we have no back end

Reply:
Anne

Reply:
that'll be fine if we add @user = current_user in the Pagescontroller#home method